sm2a/infrastructure/functions/trigger_sm2a_dag/lambda_function.py

```diff
@@ -0,0 +1,57 @@
+import http.client
+import json
+import os
+import uuid
+from base64 import b64encode
+
+import boto3
+
+
+def lambda_handler(event, context):
+    secrets_client = boto3.client("secretsmanager")
+    sm2a_secret_manager_name = os.getenv("SM2A_SECRET_MANAGER_NAME")
```

Comment:
As we iterate on this it would be nice to add config/pydantic ([example](https://github.com/NASA-IMPACT/veda-data-airflow/blob/4cf61ded2c2879c4e7f392416de4cb18686af762/workflows_api/runtime/src/config.py#L8)) to make configuration consistent across python services but since this lambda only needs the one env variable, this works.